Setup.md

```diff
@@ -0,0 +1,47 @@
+<h1 align="center"> Setup Cross Platform Activity Tracker
+ 🛠️</h1>
+
+## Step 1
+
+- Fork this repository by clicking `fork button`
+
+- Clone the repository in your local machine by typing 
+    ```
+    git clone https://github.com/<your-username>/Cross-Platform-Activity-Tracker.git
+    ``` 
+    in your terminal(for _mac/linux_) or Git Bash ( for _windows_)
+
+- Now creat a new branch using 
+    ```
+    git checkout -b <your-new-branch-name>
+    ```
+
+## Step 2
+
+- Install `nodejs` and `npm` on your local machine. For [windows](https://www.geeksforgeeks.org/installation-of-node-js-on-windows/) and [linux](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04)
+- Install `pnpm` by opening your terminal with administrative privileges and type 
+    ```
+    npm i -g pnpm
```

Comment:
You may mention the `sudo` part

```
Install `pnpm` by opening your terminal with administrative (sudo) privileges and typing
```

Reply:
> @supercoww should we include this in [CONTRIBUTING_GUIDELINES.md](https://github.com/OpenLake/Cross-Platform-Activity-Tracker/blob/main/.github/CONTRIBUTING_GUIDELINES.md) instead of creating a new file?

Yes please